docs/manuals/cli/manage-versions.md

```diff
@@ -25,9 +25,8 @@ Below is the list of all the optional arguments:
 
 ## List capabilities with the --help option
 
-To display the help with all command line arguments, you can run `taipy help manage-versions`
-command. Alternatively, you can use the `--help` or `-h` options.
-Alternatively, you can use the `--help` or `-h` options. Run `taipy manage-versions --help`
+To display the help with all command line arguments, you can run `taipy help manage-versions` command.
```

Comment:
I think that's wrong.
"To display the help with all command line arguments", you certainly don't run "taipy help manage-versions"

However, to get help on the 'manage-versions' command, you can do that indeed.

Reply:
We support both actually

Reply:
```suggestion
To display the help message of the `manage-version` command, you can run `taipy help manage-versions` command.
```